platformcomponents/desktop/content-list-item.json

```diff
@@ -0,0 +1,25 @@
+{
+  "content-list-item": {
+    "comment": "Applied to file attachments from Content tab",
+    "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=49651%3A114181",
+    "iconForeground": "@theme-common-text-white",
+    "iconBackground": {
+      "default": "@theme-background-accent-normal"
+    },
+    "#normal": {
+      "background": "@theme-background-secondary-normal",
+      "textFileName": "@theme-text-primary-normal",
```

Comment:
to be able to the UX team to put the file author (for example) instaed of the file size on that secondary label, we want to name this comment with a more generic term:
```
"label": {
       "primary": ...,
       "secondary": ....
}
```

Reply:
Pushed commit
`Organised texts into label key in content-file-list-item`
Changing this conversation to Resolved.